acquia_lift_profiles/qunit/tests.js

```diff
@@ -139,7 +139,16 @@ QUnit.asyncTest( "init test", function( assert ) {
 });
 
 
+QUnit.test("Get context values with 'do not track' cookie", function( assert ) {
+  $.cookie('tc_dnt', 'true', {path:'/'});
```

Comment:
As per our discussion, we already have two other tests on non-empty object returned (just not in the same test), so it will suffice.